config/config.go

```diff
@@ -147,12 +147,16 @@ func (c Configuration) New(
 		jaeger.TracerOptions.ZipkinSharedRPCSpan(opts.zipkinSharedRPCSpan),
 	}
 
+	for _, tag := range opts.tags {
+		tracerOptions = append(tracerOptions, jaeger.TracerOptions.Tag(tag.Key, tag.Value))
+	}
```

Comment:
hm, merge issue. You need to define the official repo as eg `upstream` and 
```
git checkout -b master
git pull upstream master
git checkout -b observer-with-spancontext
git rebase master
```
your master should be == upstream master